### DISCUSSION TITLE
Controls in Chromatic UI?

### DISCUSSION THREAD

**Discussion by ghost** [storybookjs/storybook#12614] [Help] [answered]
Not quite sure if it was ever possible, but I just recently set up a Storybook (6.0.22) project in Chromatic (5.2.0) and the canvas' screen won't show the story's controls, or accesibility tests for that matter. I have to go to the actual Storybook UI to see them, any idea if this is intended?

**Comment by shilman** [comment, marked as answer]
Please use Chromatic's on-site support chat for chromatic-related questions 🙏
